std_msgs/Float64MultiArray params
---
bool response